iFixit/react-commerce
# Troubleshooting: Fix text encoding on `RelatedProblems` card

Use `Prerendered` to properly render HTML encoded text such as `&quot;`.

QA
--
Check that text renders properly for the device card:

<details>
<summary>Image</summary>

![image](https://github.com/iFixit/react-commerce/assets/43479402/3f8ef60d-30a6-41cf-a654-5c3a5d398d8c)

</details>

Example troubleshooting page: https://me.cominor.com/Troubleshooting/iPhone_11/Will+Not+Turn+On/478192

Closes: https://github.com/iFixit/ifixit/issues/50287
